tokio-console/src/config.rs

```diff
@@ -23,16 +24,16 @@ pub struct Config {
     /// The address of a console-enabled process to connect to.
     ///
     /// This may be an IP address and port, or a DNS name.
```

Comment:
we can probably get the default values displayed in the command-line `--help` output if we just put them in the comments like this:
```suggestion
    /// This may be an IP address and port, or a DNS name.
    ///
    /// [default: http://127.0.0.1:6669/]
```
(and do the same for the other defaults)